config/install/field.field.node.islandora_object.field_table_of_contents.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_table_of_contents
+    - node.type.islandora_object
+id: node.islandora_object.field_table_of_contents
+field_name: field_table_of_contents
+entity_type: node
+bundle: islandora_object
+label: 'Table Of Contents'
+description: ''
+required: false
+translatable: true
+default_value: {  }
+default_value_callback: ''
+settings: {  }
+field_type: string_long
```

Comment:
As per [the Slack discussion](https://islandora.slack.com/archives/CM9CVQWJ0/p1579101817013800), please change to `text_long`.